.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Please, don't commit those files since they are not relevant to the project or the linked issue. Note that you can overwrite the files used by other contributors if they use PyCharm as well...

I suggest using a global `.gitignore` file in your home directory to avoid committing those files.

Reply:
I'd say it's also fine to add .idea into .gitignore of this project. Here is documentation how you can set up gitignore in your environment: https://docs.github.com/en/github/using-git/ignoring-files

Reply:
ok great thank you for this information.

Reply:
I overshot the interactive rebase too far back a few commits so I restarted again from main and made 1 commit for all changes into a new branch. Going to update via another pull as suggested. Thanks.